src/subject_name/verify.rs

```diff
@@ -295,3 +300,134 @@ impl<'a> FromDer<'a> for GeneralName<'a> {
 
     const TYPE_ID: DerTypeId = DerTypeId::GeneralName;
 }
+
+#[cfg(feature = "alloc")]
+impl fmt::Debug for GeneralName<'_> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            GeneralName::DnsName(name) => write!(
+                f,
+                "DnsName(\"{}\")",
```

Comment:
Should we include the quotes or not? I thought it made sense because domain names would usually be written as a string but on the other hand these will require escaping in lots of contexts so maybe not worth it?

Reply:
I like the quotes but don't feel strongly. If you think it's awkward it seems OK to drop them.